lib/ios/ReactNativeNavigationTests/RNNRootViewControllerTest.m

```diff
@@ -491,7 +491,7 @@ -(void)testTopBarLargeTitle_true {
 	XCTAssertEqual(self.uut.navigationItem.largeTitleDisplayMode, UINavigationItemLargeTitleDisplayModeAlways);
 }
 -(void)testTopBarLargeTitle_false {
-	self.options.tabBar.hidden = @(0);
+	self.options.bottomTabs.hidden = @(0);
```

Comment:
is this a mistake? function name is `testTopBarLargeTitle_false `

Reply:
Yeah will fix and push... weird